6_visualize/src/prep_outro_funs.R

```diff
@@ -92,15 +96,15 @@ prep_outro_readmore_fun <- function(outro_placement, legend_text_cfg, opacity=1)
 
     # plan text coordinates
     user_coords <- par()$usr
-    x_title <- user_coords[1] + 0.52 * diff(user_coords[1:2])
-    x_text <- user_coords[1] + 0.43 * diff(user_coords[1:2])
-    y_title <- user_coords[3] + 0.17 * diff(user_coords[3:4])
+    x_title <- user_coords[1] + 0.57 * diff(user_coords[1:2])
+    x_text <- user_coords[1] + 0.57 * diff(user_coords[1:2])
+    y_title <- user_coords[3] + 0.15 * diff(user_coords[3:4])
     y_text <- user_coords[3] + 0.11 * diff(user_coords[3:4])
 
     # plot text and legend
     text(x=x_title, y=y_title, labels="STAY SAFE DURING FLOODS", adj=c(0, 1),
          cex=2.2, col=legend_text_cfg$col, family = 'Oswald')
-    text_chars <- "Learn more about USGS response to Florence at www.usgs.gov/florence"
+    text_chars <- "Learn more about USGS response to Michael\nat www.usgs.gov/michael"
```

Comment:
huh, nice, that shorter URL wasn't resolving for me yesterday. guess they got it spun up!